RecoPPS/ProtonReconstruction/python/ctppsProtons_cfi.py

```diff
@@ -0,0 +1,17 @@
+import FWCore.ParameterSet.Config as cms
+
+# import default optics settings
+from CalibPPS.ESProducers.ctppsOpticalFunctions_cff import *
```

Comment:
Please make a RecoPPS/ProtonReconstruction/python/ctppsProtons_cff.py that includes both:
-  CalibPPS.ESProducers.ctppsOpticalFunctions_cff
- this  RecoPPS.ProtonReconstruction.ctppsProtons_cfi

as I suggested in https://github.com/cms-sw/cmssw/pull/42890#discussion_r1423730194 (before the "HOWEVER", as you verified that you cannot remove ctppsOpticalFunctions from it)

Reply:
I hope commit 95349b14ebd2b18765811fc25ddda8b6000620f6 answers this comment - I just didn't know where to put `import FWCore.ParameterSet.Config as cms` but as far as I understood it should be in `_cff`